open_schools_platform/common/tests/models/test_random_model.py

```diff
@@ -0,0 +1,39 @@
+from datetime import timedelta
+
+from django.test import TestCase
+from django.utils import timezone
+from django.db.utils import IntegrityError
+
+from open_schools_platform.common.models import RandomModel
+
+
+class RandomModelTests(TestCase):
```

Comment:
Send this to the docs?

Reply:
I decided to delete this test. I think this code would be useful for private docs, not public .